pyccel/ast/omp.py

```diff
@@ -76,6 +76,11 @@ def __getnewargs__(self):
         args = (self.txt, self.combined)
         return args
 
+    def __str__(self):
+        instruction = [self.name, self.combined, self.txt]
+        instruction = ' '.join(i for i in instruction if i)
+        return '#$ omp '+instruction
```

Comment:
Sorry for being pedantic. Unless strictly necessary I would like us to avoid redefining variables, especially if their type changes as a result. Here the last two lines can be merged into one, and I think that the resulting code would be clearer.

Reply:
So you prefer:
```
instruction = ' '.join(i for i in [self.name, self.combined, self.txt] if i)
```
?
I disagree that this is clearer. I can rename the list variable if you prefer?

Reply:
No, I mean:
```python
    def __str__(self):
        instruction = (self.name, self.combined, self.txt)
        return '#$ omp ' + ' '.join(i for i in instruction if i)
 ```

Reply:
And yes, the name `instruction` is not really appropriate for a tuple...

Reply:
Ah of course sorry. Fixed: 20dfcbde